src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1422,22 +1423,29 @@ private void ExecuteUninstallPackageCommand(object sender, ExecutedRoutedEventAr
                 return;
             }
 
-            UninstallPackage(package.Id);
+            UninstallPackage(package.Id, new[] { package });
         }
 
-        private void SetOptions(NuGetUI nugetUi, NuGetActionType actionType)
+        private void SetOptions(NuGetUI nugetUi, NuGetActionType actionType, IEnumerable<PackageItemViewModel> packages)
         {
             var options = _detailModel.Options;
+            IEnumerable<PackageItemViewModel> vulnerablePkgs = packages?
+                .Where(x => x.Vulnerabilities?.Any() ?? false) ??
+                Enumerable.Empty< PackageItemViewModel>();
```

Comment:
Extra space after '<' in Enumerable.Empty< PackageItemViewModel>();

Reply:
Fixed.